### PR TITLE
Replace raw memsets and memcpys

### PR DESCRIPTION
**Pull request by yebblies**

### CODE REVIEW

File: src/scanmscoff.d [hunk 1]
```diff
@@ -49,7 +49,7 @@ extern (C++) void scanMSCoffObjModule(void* pctx, void function(void* pctx, char
         header_old = cast(IMAGE_FILE_HEADER*)malloc(IMAGE_FILE_HEADER.sizeof);
         memcpy(header_old, buf, IMAGE_FILE_HEADER.sizeof);
         header = cast(BIGOBJ_HEADER*)malloc(BIGOBJ_HEADER.sizeof);
-        memset(header, 0, BIGOBJ_HEADER.sizeof);
+        header = BIGOBJ_HEADER.init;
```
Comment (MartinNowak) [inline src/scanmscoff.d]: `*header = BIGOBJ_HEADER.init`

**Comment by MartinNowak** [comment]
Looks like you activated `-vgc` by default.

**Comment by WalterBright** [comment]
LGTM except for the -vgc errors.

**Comment by yebblies** [comment]
Of course, the errors are because `char` is used to store small integers in some places.

**Comment by MartinNowak** [comment]
Ping

**Comment by yebblies** [comment]
Rebased and fixed whitespace errors

**Comment by 9rnsr** [comment]
Auto-merge toggled on

**Pull request merged**